modules/sr/robot/ruggeduino_devices.py

```diff
@@ -43,6 +43,16 @@ class Led:
 
     def __init__(self, webot, device_name):
         self.webot_sensor = webot.getLED(device_name)
+        self._webot = webot
```

Comment:
nit: If we're keeping reference to `webot`, might as well use it everywhere

```suggestion
        self._webot = webot
        self.webot_sensor = self._webot.getLED(device_name)
```

Reply:
I'm not sure what benefit this provides? I think either spelling is fine, so I'd rather err on the side of leaving existing code as-is.